setup.py

```diff
@@ -1,19 +1,21 @@
 #!/usr/bin/env python
 
 import os
+import setuptools  # noqa
 
-import setuptools
 from distutils.core import setup
 
-execfile('grouper/version.py')
+from grouper.version import __version__
```

Comment:
ah, ok. well, i'll revert the behavior. maybe need to exclude `setup.py` from flake8 or just `# noqa` that line that references `__version__` then.